tests/scripts/all.sh

```diff
@@ -2105,6 +2105,7 @@ config_psa_crypto_config_ecdsa_use_psa () {
     # TODO: make these work - #6862
     scripts/config.py unset MBEDTLS_KEY_EXCHANGE_ECDHE_ECDSA_ENABLED
     scripts/config.py unset MBEDTLS_KEY_EXCHANGE_ECDH_ECDSA_ENABLED
+    scripts/config.py unset MBEDTLS_ECP_RESTARTABLE
```

Comment:
Minor: perhaps add a comment above that this isn't expected to work for now (mostly to clarify that the comment 3 lines above only applies to the key exchange lines, not this one).